Makefile

```
@@ -51,3 +51,11 @@ dabl: dabl-ui
 
 dabl-ui: build
 	$(MAKE) -C ui dabl-ui LEDGER_ID=$(LEDGER_ID)
+
+DAR_FILES=$(shell find . -type f -name "*.dar")
+
+package: dabl
```

not too important, merely a suggestion (feel free to ignore this)
it's just `package` sounds like a useful goal to our users. however, it's internal, we should frighten them a bit not to use this.
```suggestion
dabl-package: dabl
```

---

Done